lib/chunks.js

```diff
@@ -173,17 +173,29 @@ module.exports.chunkMetadataFromDatabaseChunk = function (chunk) {
         type: chunk.type,
       };
     case 'clientFilesCourseInstance':
+      if (!chunk.course_instance_name) {
+        throw new Error(`course_instance_name is missing for chunk ${chunk.uuid}`);
+      }
```

Comment:
We could likely avoid all these checks by defining type-specific versions of `DatabaseChunk` (e.g. `QuestionDatabaseChunk`, `ElementsDatabaseChunk`, etc.) and validating with Zod when fetching from the DB. I went with the simpler option for now though.

Reply:
I've been increasingly using Union types and really enjoying them, but I'm fine with the approach of deferring this for now.

Reply:
Yeah, I love them too, and make extensive use of them in this file! I'm just trying to KISS a bit for now.